README.md

```diff
@@ -47,8 +72,12 @@ Report issues for this extension in the [ocds-extensions repository](https://git
 
 ## Changelog
 
+### 2020-04-29
+
+* Add a`deliveryAddresses` array to the item object to specify the delivery addresses.
```

Comment:
```markdown
* Add `Item.deliveryAddresses` field
```

I prefer to leave out the field descriptions, in case we update them in the schema.

Other changelogs omit the "a" article and period, so we can leave them out here.

Reply:
Done

Reply:
Hmm, missing `Item.`

Reply:
Done